apps/tasks/src/App.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <Molgenis id="__top" v-model="session">
```

Comment:
__top ?

Reply:
ah, that is a copy/paste artifact from an app that does scrolling and then needed 'top' anchor. Remove.